src/actions/accountsActions.js

```diff
@@ -54,12 +55,7 @@ import { setActiveBlockchainNetworkAction } from './blockchainNetworkActions';
 const storage = Storage.getInstance('db');
 
 export const initDefaultAccountAction = (walletAddress: string, walletId: string, migrateData: boolean = true) => {
-  return async (dispatch: Dispatch) => {
-    const { balances = {} } = await storage.get('balances');
-    const { history = {} } = await storage.get('history');
-    const { collectibles = {} } = await storage.get('collectibles');
-    const { collectiblesHistory = {} } = await storage.get('collectiblesHistory');
-
+  return async (dispatch: Function) => {
```

Comment:
Why are we removing the type here?

Reply:
Sorry, that was made unintentionally. I've put it back :)